packages/client/hmi-client/src/App.vue

```diff
@@ -84,7 +84,7 @@ import Button from 'primevue/button';
 import { ToastSummaries, ToastSeverity, useToastService } from '@/services/toast';
 import { useRoute, useRouter } from 'vue-router';
 import API from '@/api/api';
-import Navbar from '@/components/Navbar.vue';
+import TeraNavbar from '@/page/data-explorer/components/navbar/tera-navbar.vue';
```

Comment:
Why tera-navbar is being moved with the data-explorer?
It's use by App.vue, it should stay within component.